config/initializers/paperclip.rb

```diff
@@ -1,6 +1,6 @@
 Paperclip::Attachment.default_options.merge!(
   path: ":rails_root/uploads/:class/:attachment/:id_partition/:style/:filename",
-  url: "/:locale/m/:mission/:class/:id/:style"
+  url: "/:locale/m/:mission/:class/:attachment/:id/:style"
```

Comment:
This is actually not being used now, right? It can't be because it's incorrect!

This only gets used if we do `question.audio_prompt.url` but we're not doing that. So I'd say delete this line.